jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants.properties

```diff
@@ -22,6 +22,7 @@ Due_On=Due On
 Open=Open
 Release=Release
 Claim=Claim
+ClaimAndWork=Claim and Work
```

Comment:
@bxf12315 @nmirasch @jstastny-cz this is a new action in the dropdown that claims the task and open the details.
Are you guys happy with _Claim and Work_ or should it be something more abvious like _Claim and View/Go to Details_?

Reply:
LGTM